app/src/main/java/com/sopt/kakaobank/presentation/home/HomeRecyclerViewItemDecorator.kt

```diff
@@ -0,0 +1,23 @@
+package com.sopt.kakaobank.presentation.home
+
+import android.content.Context
+import android.graphics.Rect
+import android.util.TypedValue
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+
+class HomeRecyclerViewItemDecorator(context: Context) : RecyclerView.ItemDecoration() {
```

Comment:
```suggestion
class ~ItemDecorator(val context: Context) : RecyclerView.ItemDecoration() {
    override fun getItemOffsets(
        outRect: Rect,
        view: View,
        parent: RecyclerView,
        state: RecyclerView.State
    ) {
        super.getItemOffsets(outRect, view, parent, state)
        val position = parent.getChildAdapterPosition(view)

        if (position == 0) {
            outRect.top = context.pxToDp(16)
            outRect.bottom = context.pxToDp(12)
        } else {
            outRect.bottom = context.pxToDp(12)
        }
    }
}
```
저는 보통 이런식으로 많이 구현해요!